lib/scsi_defs.h

```diff
@@ -803,6 +803,53 @@ struct UnmapBlockDescriptor {
 } ABSL_ATTRIBUTE_PACKED;
 static_assert(sizeof(UnmapBlockDescriptor) == 16);
 
+// SCSI Reference Manual Table 483
+// https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+struct SupportedVitalProductData {
+  PeripheralQualifier peripheral_qualifier : 3;
+  PeripheralDeviceType peripheral_device_type : 5;
+  uint8_t page_code : 8;
+  uint8_t _reserved : 8;
+  uint8_t page_length : 8;
+  uint8_t supported_page_list[256];
+} ABSL_ATTRIBUTE_PACKED;
```

Comment:
nit: the code in master does static asserts for size. might be good to have that here for safety/consistency